### PR TITLE
POWR-1260 Update Employees CI script

### PR DESCRIPTION
**Pull request by kxwang**

### CODE REVIEW

File: xdebug.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 variables_order = "EGPCS"
-xdebug.remote_host = 10.0.75.1
+xdebug.remote_host = ${LANDO_HOST_IP}
```
Comment (mike-mcdonald) [inline xdebug.ini]: I'd rather this be 'host.docker.internal' but this is fine.  See https://docs.docker.com/docker-for-mac/networking/#use-cases-and-workarounds

File: pantheon.yml [hunk 2]
```diff
@@ -26,15 +26,42 @@ workflows:
...
     after:
+      - type: webphp
+        description: Rebuild cache before config import
+        script: private/scripts/drush_post_deploy_cache_rebuild.php
+      - type: webphp
+        description: Update database
+        script: private/scripts/drush_post_deploy_updatedb.php
+      - type: webphp
+        description: Import configuration from .yml files
+        script: private/scripts/drush_post_deploy_config_import.php
```
Comment (mike-mcdonald) [inline pantheon.yml]: Maybe we should move a cache rebuild here, after this script too?

File: pantheon.yml [hunk 3]
```diff
@@ -26,15 +26,42 @@ workflows:
...
+      - type: webphp
+        description: Rebuild cache before config import
+        script: private/scripts/drush_post_deploy_cache_rebuild.php
+      - type: webphp
+        description: Update database
+        script: private/scripts/drush_post_deploy_updatedb.php
       - type: webphp
         description: Import configuration from .yml files
-        script: private/scripts/drush_post_deploy.php
+        script: private/scripts/drush_post_deploy_config_import.php
```
Comment (mike-mcdonald) [inline pantheon.yml]: Maybe we should move a cache rebuild here, after this script too?

File: xdebug.ini (hunk 1 above)
Comment (kxwang) [reply xdebug.ini]: I didn't know that. Thanks for the info. I found in Lando document "LANDO_HOST_IP=host.docker.internal".

File: pantheon.yml (hunk 2 above)
Comment (kxwang) [reply pantheon.yml]: Will add a CR here.

File: pantheon.yml (hunk 3 above)
Comment (kxwang) [reply pantheon.yml]: Will add a CR here

**Comment by kxwang** [comment]
@mike-mcdonald I've updated pantheon.yml based on your input. But the commit triggered another request for review. This one should be simple. Thx!

**Pull request merged**